src/particles/particles_3D.cpp

```diff
@@ -176,8 +182,16 @@ void Particles3D::Initialize(struct Parameters *P, Grav3D &Grav, Real xbound, Re
     #endif
 
   G.size_blocks_array = 1024 * 128;
-  G.n_cells_potential = (G.nx_local + 2 * N_GHOST_POTENTIAL) * (G.ny_local + 2 * N_GHOST_POTENTIAL) *
-                        (G.nz_local + 2 * N_GHOST_POTENTIAL);
+  {
```

Comment:
Why is there a code block here?  Is it to limit the scope of `int n_ghost_pot`?  I'm only curious, I'm not proposing any changes.

Reply:
Oh, it's because this is a fairly large function with a bunch of local variables, and it is already difficult to reason about all of the local variables. I was just trying to localize `n_ghost_pot` to a single area